modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/session/SessionManagerTest.java

```diff
@@ -0,0 +1,128 @@
+package org.apache.ignite.internal.sql.engine.session;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.Map;
+import java.util.UUID;
+import javax.validation.constraints.AssertTrue;
+import org.apache.ignite.internal.sql.engine.property.PropertiesHolder;
+import org.apache.ignite.internal.sql.engine.property.Property;
+import org.apache.ignite.internal.testframework.IgniteTestUtils;
+import org.jetbrains.annotations.Nullable;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class SessionManagerTest {
+
+    private SessionManager sessionMgr;
+    private Map<SessionId, Session> activeSessions;
+
+    @BeforeEach
+    void beforeEach() {
+        sessionMgr = new SessionManager("test", System::currentTimeMillis);
+        activeSessions = IgniteTestUtils.getFieldValue(sessionMgr, "activeSessions");
+    }
+
+    @AfterEach
+    void afterEach() throws Exception {
+        sessionMgr.stop();
+    }
+
+    @Test
+    void createSession() {
```

Comment:
I'm not sure this test is useful. Why do you think we need to verify that size of some internal collection correlates to amount of invocation of some method? 

To me, the only thing matters here is that you are able to retrieve session by returned identifier

Reply:
removed